config.json

```diff
@@ -481,7 +481,7 @@
       "unlocked_by": "luhn",
       "difficulty": 4,
       "topics": [
-        "function_pointer"
+        "generics", "higher_order_functions", "move_semantics"
```

Comment:
Because you've changed config.json, you'll need to rerun `configlet fmt` in order for Travis to succeed.